src/utils/bcrypt.ts

```diff
@@ -0,0 +1,7 @@
+import * as bcrypt from 'bcrypt';
+
+export const encodePassword = (rawPassword: string) => {
```

Comment:
**praise**: The encodePassword function is well-implemented for hashing passwords with a salt.